src/components/List/VideoList.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { v4 } from 'uuid';
+import Styled from './VideoListStyled';
+import { items } from '../../mock-data.json';
+
+class VideoList extends Component {
```

Comment:
this is fine but I think you would enjoy using functional components instead of class components